### PR TITLE
[Android] Show username on profile button in settings

### PR DESCRIPTION
**Pull request by RedAuburn**
fixes https://github.com/organicmaps/organicmaps/issues/5549

image1

### CODE REVIEW

**Comment by RedAuburn** [comment]
@organicmaps/translations please check translations in `[Android] show username on profile button in settings` :)

File: data/strings/strings.txt [hunk 1]
```diff
@@ -13692,6 +13692,51 @@
...
+    cs = Nepřihlášen
+    da = Ikke logget ind
+    de = Nicht eingeloggt
+    el = Δεν έχετε συνδεθεί
+    es = No iniciado sesión
+    et = Ei ole sisse logitud
+    eu = Saioa hasi gabe
+    fa = وارد نشده است
+    fi = Ei kirjautunut sisään
+    fr = Non signé
```
Comment (Arthur-GYT) [inline data/strings/strings.txt:13713]: For French, it would be "Non connecté•e" fir gebder neutral ^^

**Review by d4f5409d — APPROVED** [review]
🇭🇺 hu seems good

**Comment by Jean-BaptisteC** [comment]
Is it really necessary to see a subtitle when the user is not connected?
If we consider most of our users have an OSM account, this text is never visible.

**Comment by RedAuburn** [comment]
> Is it really necessary to see a subtitle when the user is not connected? If we consider most of our users have an OSM account, this text is never visible.

No harm in having it IMO, it'll remind people that don't have an account that they could get one, sort of like a reverse dark pattern 😉

File: data/strings/strings.txt [hunk 2]
```diff
@@ -13692,6 +13692,51 @@
...
+    ar = لم يتم تسجيل الدخول
+    az = Daxil deyil
+    be = Не ўвайшлі
+    bg = Не е влязъл в системата
+    ca = No s'ha iniciat la sessió
+    cs = Nepřihlášen
+    da = Ikke logget ind
+    de = Nicht eingeloggt
+    el = Δεν έχετε συνδεθεί
+    es = No iniciado sesión
```
Comment (dvdmrtnz) [inline data/strings/strings.txt:13708]: Spanish sounds a bit weird. I would say "No se ha iniciado sesión" or "Sesión no iniciada" CC @organicmaps/translations-es

**Comment by NeatNit** [comment]
Hebrew (`he`) is good.

File: data/strings/strings.txt [hunk 3]
```diff
@@ -13692,6 +13692,51 @@
...
+    it = Non è stato effettuato l'accesso
+    ja = サインインしていない
+    ko = 로그인하지 않음
+    lt = Neprisijungta
+    mr = साइन इन नाही
+    nb = Ikke logget inn
+    nl = Niet aangemeld
+    pl = Nie zalogowano się
+    pt = Não estás inscrito
+    pt-BR = Não conectado
```
Comment (matheusgomesms) [inline data/strings/strings.txt:13727]:
```suggestion
    pt = Sessão não iniciada
```

File: data/strings/strings.txt (hunk 1 above)
Comment (RedAuburn) [reply data/strings/strings.txt:13713]: thanks 👍️

File: data/strings/strings.txt (hunk 2 above)
Comment (RedAuburn) [reply data/strings/strings.txt:13708]: thanks 👍️

File: data/strings/strings.txt [hunk 4]
```diff
@@ -13692,6 +13692,51 @@
...
+    en = Not signed in
+    af = Nie aangemeld nie
+    ar = لم يتم تسجيل الدخول
+    az = Daxil deyil
+    be = Не ўвайшлі
+    bg = Не е влязъл в системата
+    ca = No s'ha iniciat la sessió
+    cs = Nepřihlášen
+    da = Ikke logget ind
+    de = Nicht eingeloggt
```
[the remark this answers is not in the document]
Comment (RedAuburn) [reply data/strings/strings.txt:13706]: thanks 👍️

File: data/strings/strings.txt (hunk 3 above)
Comment (RedAuburn) [reply data/strings/strings.txt:13727]: thanks 👍️

**Comment by RedAuburn** [comment]
Strings updated, and No longer relies on the other PR, PTAL @biodranik @vng

**Comment by biodranik** [comment]
Is there a way to display the login name on the right? E.g.

    OpenStreetMap profile                     RedAuburn

In this case translations won't be needed. Current translations for some languages are not good...

**Comment by RedAuburn** [comment]
> Is there a way to display the login name on the right? In this case translations won't be needed. 

It's definitely possible; however, I do think it makes more sense to set it in the summary.
1. Really long names may wrap in a strange way
2. It aligns more with material guidelines/android UI practices to put this info in the summary

> Current translations for some languages are not good...

in that case it's better to fix them, IMO :)

are these better?
```
[not_signed_in]
    tags = android
    en = Not signed in
    af = Nie ingeteken nie
    ar = لم يتم تسجيل الدخول
    az = Giriş edilməyib
    be = Не ўвайшлі ў сістэму
    bg = Не е влязъл в системата
    ca = No heu iniciat sessió
    cs = Nejste přihlášeni
    da = Ikke logget ind
    de = Nicht angemeldet
    el = Δεν έχετε συνδεθεί
    es = No se ha iniciado sesión
    et = Pole sisse logitud
    eu = Ez zaude saiatuta
    fa = وارد نشده‌اید
    fi = Et ole kirjautunut sisään
    fr = Non connecté•e
    he = לא מחובר
    hi = साइन इन नहीं किया गया है
    hu = Nincs bejelentkezve
    id = Belum masuk
    it = Non sei connesso
    ja = サインインしていません
    ko = 로그인되지 않음
    lt = Neprisijungta
    mr = साइन इन केलेले नाही
    nb = Ikke logget inn
    nl = Niet ingelogd
    pl = Nie zalogowano
    pt = Sessão não iniciada
    ro = Neautentificat
    ru = Не выполнен вход
    sk = Nie ste prihlásený
    sv = Inte inloggad
    sw = Hujasajiliwa
    th = ยังไม่ได้ลงชื่อเข้าใช้
    tr = Giriş yapılmadı
    uk = Не ввійшли
    vi = Chưa đăng nhập
    zh-Hans = 未登录
    zh-Hant = 未登入
```

**Review by Arthur-GYT — APPROVED** [review]
LGTM

**Comment by biodranik** [comment]
> are these better?

Yes, these are slightly better.

**Comment by RedAuburn** [comment]
Now using the better strings. Merge? @biodranik

File: data/strings/strings.txt [hunk 5]
```diff
@@ -13692,6 +13692,50 @@
     zh-Hans = 登录
     zh-Hant = 登入
 
+  [not_signed_in]
+    tags = android
```
Comment (biodranik) [inline data/strings/strings.txt:13696]: comment = Status message indicating that user did not login to OSM profile yet.

**Review by biodranik — APPROVED** [review]
Thanks! LGTM with two nits.

**Comment by RedAuburn** [comment]
nits fixed @biodranik :)

**Comment by biodranik** [comment]
Thanks! :)

**Pull request merged**